VK_V1.0/VK_V1.0/MyGroups/Model/Group.swift

```diff
@@ -0,0 +1,10 @@
+// Group.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
```

Comment:
```suggestion
import Foundation
```
не супер важно но тут можно убирать импорты